src/services/tags-service.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * @file Implements the service for tuits
+ */
+import axios from "axios";
+const BASE_URL = process.env.REACT_APP_BASE_URL;
+const TAGS_API = `${BASE_URL}/api/tags`;
+
+/**
+ * Since tags-service interacts with an API that uses session, axios is
+ * configured to include cookies in the headers to support credentials.
+ * @type {AxiosInstance}
+ */
+export const api = axios.create({
+                                    withCredentials: true
+                                });
+
+/**
+ * Uses API to find all tags.
+ * @returns {Promise<AxiosResponse<any>>} JSON array with tags
+ */
+export const findAllTags = () =>
+    api.get(TAGS_API)
+        .then(response => response.data);
+
+/**
+ * Uses API to retrieve a tags by its primary key.
+ * @param tid Primary key of tag
```

Comment:
Let's change this to tagID also.

Reply:
I'm just going to go ahead and get rid of this function if that's good with you. I don't think we'll need to call findTagById from the front end.

Reply:
Works for me!